web/app/loader.js

```
@@ -27,162 +28,154 @@ window.Progressive = {}
-        // like incognito mode, in which case there is no point getting
-        // Messaging worker version data, we can just use the base URL.
-        if (!isLocalStorageAvailable()) {
-            return SW_LOADER_PATH
-        }
-
-        const workerPathElements = [SW_LOADER_PATH]
-
-        const swVersion = getMessagingSWVersion()
-        if (swVersion) {
-            workerPathElements.push(`msg_sw_version=${swVersion}`)
-        }
+    const workerPathElements = [SW_LOADER_PATH]
 
-        // Return the service worker path
-        return workerPathElements.join('&')
+    const swVersion = getMessagingSWVersion()
+    if (swVersion) {
+        workerPathElements.push(`msg_sw_version=${swVersion}`)
     }
 
-    /**
-     * Load the service worker
-     * @returns Promise.<Boolean> true when the worker is loaded and ready
-     */
-    const loadWorker = () => (
-        navigator.serviceWorker.register(getServiceWorkerURL())
-            .then(() => navigator.serviceWorker.ready)
-            .then(() => true)
-            .catch(() => {
-            })
-    )
-
-    const asyncInitApp = () => {
-        window.webpackJsonpAsync = (module, exports, webpackRequire) => {
-            const runJsonpAsync = () => {
-                if (window.webpackJsonp) {
-                    // Run app
-                    window.webpackJsonp(module, exports, webpackRequire)
-                } else {
-                    setTimeout(runJsonpAsync, 50)
-                }
-            }
+    // Return the service worker path
+    return workerPathElements.join('&')
+}
 
-            runJsonpAsync()
+/**
+ * Load the service worker
+ * @returns Promise.<Boolean> true when the worker is loaded and ready
+ */
+const loadWorker = () => (
+    navigator.serviceWorker.register(getServiceWorkerURL())
+        .then(() => navigator.serviceWorker.ready)
+        .then(() => true)
+        .catch(() => {
+        })
```

I know this isn't your code, but empty `catch()` blocks in any language always look suspect. We should have a comment here that at least says we're swallowing errors here on purpose.